airbyte-config/init/src/main/resources/seed/source_definitions.yaml

```diff
@@ -1676,14 +1676,6 @@
   icon: qualaroo.svg
   sourceType: api
   releaseStage: alpha
-- name: QuickBooks
-  sourceDefinitionId: 29b409d9-30a5-4cc8-ad50-886eb846fea3
-  dockerRepository: airbyte/source-quickbooks-singer
-  dockerImageTag: 0.1.5
-  documentationUrl: https://docs.airbyte.com/integrations/sources/quickbooks-singer
-  icon: qb.svg
-  sourceType: api
-  releaseStage: alpha
```

Comment:
This looks like you are just removing the connector?

Reply:
I'm replacing a singer based Quickbooks with a lowcode-based one. The seed for the new one should be generated after I publish the connector. So the old one is removed, am I doing it the wrong way?